contrib/bmp2/README.md

```diff
@@ -0,0 +1,38 @@
+BMP2
+====
+
+![BMP2](http://ebrombaugh.studionebula.com/embedded/bmp2/bmp2.jpg)
+
+#### BMP2 assembled board
+
+### What is it?
+
+This is a revised layout for the Black Magic Probe Mini from Blacksphere which provides connections for several styles of JTAG connectors. In addition to the 10-pin mini JTAG found on the original, it also has connections compatible with the original 20-pin 0.1" pitch JTAG connector as well as the ST-Link 6-pin 0.1" single row header.
+
+Other than the addition of these connectors and a revised layout that's more suited for DIY assembly, nothing has been changed. Original BMPM firmware is 100% compatible.
+
+### Resources
+
+*   [Black Magic Probe (BMP)](http://www.blacksphere.co.nz/main/blackmagic) Original BMP web site.
```

Comment:
This website does not exist any more. The official website is https://black-magic.org. https://black-magic.org/